OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/impl/OperationRepo.kt

```diff
@@ -69,6 +69,10 @@ internal class OperationRepo(
     private val executeBucket get() =
         if (enqueueIntoBucket == 0) 0 else enqueueIntoBucket - 1
 
+    /**
+     * Perform initialization in background to prevent possible operation point of failures from
+     * blocking the creation of OperationRepo.
+     */
```

Comment:
It would make more sense if this sort of comment was located around `loadSavedOperations()` in the code.